### PR TITLE
Fix typo in introduction.adoc

### PR DESCRIPTION
**Pull request by eslep**
Hi there! Wanted to propose a fix for what appeared to be a small typo I found while reading :)

Feel free to close if I misunderstood what was meant here -- but then maybe something is unclear and could use further clarification in the doc?

ie. should it be 'Our APIs most-purely express' or 'Our APIs must purely express'? I could understand either from the context, but the meaning is different.

### CODE REVIEW

File: chapters/introduction.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Zalando’s software architecture centers around decoupled microservices
 that provide functionality via RESTful APIs with a JSON payload. Small
 engineering teams own, deploy and operate these microservices in their
-AWS (team) accounts. Our APIs most purely express what our systems do,
+AWS (team) accounts. Our APIs must purely express what our systems do,
```
Comment (tfrauenstein) [inline chapters/introduction.adoc:7]:
```suggestion
AWS (team) accounts. Our APIs express most purely what our systems do,
```

**Comment by tfrauenstein** [comment]
Thx. for the PR! We wanted to formulate a statement not a guideline here.
Provided a suggestion to (hopefully) make it clearer.

**Comment by eslep** [comment]
@tfrauenstein Thanks for the clarification! I've committed your suggestion now

**Comment by tkrop** [comment]
👍

**Pull request merged**